### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by masnax**
Both MicroCeph & MicroOVN have updated to the most recent microcluster, so MicroCloud should as well.

Additionally, this fixes the issue with MicroCloud no longer working with `latest/edge` MicroOVN and MicroCeph due to error messages changing.

### CODE REVIEW

File: api/types/version.go [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package types
+
+// APIVersion is the current API version.
+const APIVersion = "1.0"
```
Comment (roosterfish) [inline api/types/version.go:4]: Should this be set to the latest MicroCloud release `1.1`?

**Review by roosterfish — CHANGES_REQUESTED** [review]
One comment on the version rest looks good.

It also requires a version update to `1.22.4` in the `Check compatibility with min Go version (1.22.3)` action.

**Comment by simondeziel** [comment]
> It also requires a version update to `1.22.4` in the `Check compatibility with min Go version (1.22.3)` action.

In LXD, the [`Makefile`](https://github.com/canonical/lxd/blob/main/Makefile#L14) defines `GOMIN` which is then used in the [CI job](https://github.com/canonical/lxd/blob/main/.github/workflows/tests.yml#L245-L249)

File: api/types/version.go (hunk 1 above)
Comment (masnax) [reply api/types/version.go:4]: I don't believe we are making any changes to the MicroCloud API in this release so no.

Comment (masnax) [reply api/types/version.go:4]: This is the API version that is in the path prefix: like `127.0.0.1:9000/1.0`

Comment (roosterfish) [reply api/types/version.go:4]: True, I somehow mixed it with the versioning @MggMuggins is currently working on in https://github.com/canonical/microcluster/pull/165.

**Review by roosterfish — DISMISSED** [review]
LGTM!

**Review by simondeziel — DISMISSED** [review]

**Comment by masnax** [comment]
@simondeziel Did I miss something? Looks like the `Code` workflow isn't returning?

**Comment by simondeziel** [comment]
> @simondeziel Did I miss something? Looks like the `Code` workflow isn't returning?

Just looked at it and it started "moving" and has made progress now. It was stuck in the `setup-go` action for a while.

**Comment by masnax** [comment]
> > @simondeziel Did I miss something? Looks like the `Code` workflow isn't returning?
> 
> Just looked at it and it started "moving" and has made progress now. It was stuck in the `setup-go` action for a while.

Oh, that was me, sorry for the confusion. I re-ran the `Code` block but it seems to have gotten stuck again.

**Review by simondeziel — APPROVED** [review]

**Review by roosterfish — APPROVED** [review]

**Pull request merged**